src/status_im/multiaccounts/login/flow_test.cljs

```diff
@@ -10,28 +10,28 @@
 (deftest on-password-input-submitted
   (testing
     "handling :multiaccounts.login.ui/password-input-submitted event"
-    (let [cofx {:db {:multiaccounts/login {:key-uid  "key-uid"
-                                           :password "password"
-                                           :name     "user"}}}
+    (let [cofx {:db {:profile/login {:key-uid  "key-uid"
+                                     :password "password"
+                                     :name     "user"}}}
           efx  (login.core/login cofx)]
       (testing "Change multiaccount."
         (is (= (::login.core/login efx)
                ["key-uid" "{\"name\":\"user\",\"key-uid\":\"key-uid\"}"
                 (ethereum/sha3 "password")])))
       (testing "start activity indicator"
-        (is (= (get-in efx [:db :multiaccounts/login :processing]) true))))))
+        (is (= (get-in efx [:db :profile/login :processing]) true))))))
 
 (deftest login-success
   (testing ":accounts.login.callback/login-success event received."
-    (let [db       {:multiaccounts/login {:address  "address"
-                                          :password "password"}
-                    :multiaccount        data/multiaccount}
+    (let [db       {:profile/login   {:address  "address"
+                                      :password "password"}
+                    :profile/profile data/multiaccount}
           cofx     {:db db}
           efx      (login.core/multiaccount-login-success cofx)
           json-rpc (into #{} (map :method (:json-rpc/call efx)))]
       ;; TODO: Account is now cleared only after all sign in fx are executed.
```

Comment:
nit, but perhaps you know. What is this TODO referring to? is it possible we can remove it or add an issue for it instead? 🤔

Reply:
idk, i can take a look later when i'll be moving this namespace, it will be in next PR